### PR TITLE
OKTA-611855 - Add OIN configuration doc warning example

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add note and example to the customer configuration doc: Big Bang configuration warning
- **Is this PR related to a Monolith release?** No

### Preview:
https://646f6b256c46a1194f80f174--reverent-murdock-829d24.netlify.app/docs/guides/submit-app-prereq/main/#configuration-steps

### Resolves:

* [OKTA-611855](https://oktainc.atlassian.net/browse/OKTA-611855)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/9aa492c41ab758c92fd6c99fce3fe2836aaeadbb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c1463ca5-d46b-420b-a174-e942eb982d80) | 50  | 88  | 67  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.49a66e64-3a8e-4d8e-85a9-e562dacd5a5a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md [hunk 1]
```diff
@@ -222,6 +222,25 @@ For more information on the listed features, visit the [Okta Glossary](https://h
 
 This section helps you define how your customers get set up with your integration. Detail all settings and include any images that can assist the user. Include any best practices for your procedure, such as SCIM guidance on mappings for attributes, especially required attributes that don't have a default mapping.
 
+> **Note:** If you configure Okta as a Big Bang Service Provider, you need to provide a warning note to your customer. See [SAML configuration warning example](#saml-configuration-warning-example).
```
Comment (ykrestyanov-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:225]: In this case Okta is not Service provider, please update this sentence to:
**Note:** If your Service Provider configured as a "Big Bang", you need to provide a warning note to your customer. See [SAML configuration warning example](#saml-configuration-warning-example).

**Review by ykrestyanov-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:225]: @ykrestyanov-okta: Thank you, I'll update this. 
Does L229 make sense then? Can you please review?

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/3ecfa40a955ef491236e353b6a9f9080e4174c18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f8ff005d-0d56-4135-9ead-66ae56b8378e) | 50  | 88  | 67  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b10feb11-0d39-41f9-9a95-432f5172c1be?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/34849efebefb529167bdedd2e0abec9f95cf5ac0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c8d9b229-23cf-42c3-9684-b0c4b13e59a4) | 50  | 88  | 67  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.185ad2d1-7826-4655-a202-0a36b132d1c4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/7d13e65ac688338a8c8692696669c00961e9e3a9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8144bbd2-e8f1-4b30-a624-4ed8df03d588) | 50  | 88  | 67  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9fe2abf8-c182-471b-9db7-0c2475cec302?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ykrestyanov-okta — APPROVED** [review]

**Comment by pvershinin-okta** [comment]
look good to me!

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**